reana-ui/package.json

```diff
@@ -16,19 +16,23 @@
     "react-minimal-pie-chart": "^8.0.1",
     "react-redux": "^7.1.1",
     "react-router-dom": "^5.1.2",
-    "react-scripts": "^4.0.0",
+    "react-scripts": "^5.0.0",
     "redux": "^4.0.4",
     "redux-devtools-extension": "^2.13.8",
     "redux-thunk": "^2.3.0",
-    "semantic-ui-css": "^2.4.1",
-    "semantic-ui-react": "^0.88.2"
+    "semantic-ui-react": "^2.0.0"
   },
   "scripts": {
     "analyze": "source-map-explorer 'build/static/js/*.js'",
     "start": "craco start",
     "build": "craco build",
     "test": "craco test --env=jsdom",
-    "eject": "craco eject"
+    "lint": "eslint .",
+    "prettier": "prettier --check . || (echo \"oops 🙀 pls run 'yarn fmt'\" && exit 1)",
+    "fmt": "prettier --write .",
+    "ci": "run-p lint prettier",
+    "eject": "craco eject",
+    "postinstall": "semantic-ui-css-patch"
```

Comment:
Lots of new commands were added, it's not completely clear for me when one has to use it? For example `postinstall` command is triggered automatically or one has to do it at some point? Maybe it would be worth to have a short description in the readme?

Reply:
https://docs.npmjs.com/cli/v8/using-npm/scripts#npm-install

Reply:
Would something like this make sense?

```
Yarn scripts
============

- `start`: start a development server
- `build`: build a production-ready bundle in the `build` folder
- `test`: run unit tests
- `lint`: run linter
- `prettier`: check code formatting with prettier
- `fmt`: fix formatting problems with prettier
- `ci`: run both linter and format checkers, useful before committing changes
```
Most of them are very repetitive but it might help, not sure, let me know.

Reply:
looks nice and clear in my opinion! thanks